lib/panda/http_request.rb

```diff
@@ -39,10 +36,14 @@ def headers
       )
     end
 
+    def method
+      raw_method.downcase.to_sym
+    end
+
     private
 
     def get?
```

Comment:
I'm not sure if we use this method, maybe remove it?

Reply:
We are using it [here](https://github.com/revealbot/panda/pull/11/files#diff-d3ffd6c8edaecf206f95be9eb65a923fb56cd10c99ed8153649731400323c21cR28)

Reply:
I'm talking about method `get?`, I think we can remove it

Reply:
I think method `get?` was added because of [this case](https://github.com/revealbot/panda/blob/main/lib/panda/http_request.rb#L28)

Reply:
Yeah, you're right, Github search sucks 🤦‍♂️ 
<img width="503" alt="image" src="https://github.com/user-attachments/assets/895637e4-9e17-48c5-a75b-c8b9357e2658" />